R/PlatinumGenomes-QC/Sample-Level-QC.md

```diff
@@ -50,6 +51,82 @@ pca <- read.table("./data/platinum-genomes-X-1kg-brca1-pca.tsv",
 # source("./rHelpers/pgpCGIOnlyDataset.R")
 ```
 
+## Genome Call Rate
+
+For each genome, count the number of calls (both variants and non-variant calls, if applicable).  Any genomes whose count is far away from the mean may indicate a problem.
```

Comment:
"may indicate a problem": is it worth being clear "may indicate a sample quality problem"? Otherwise "a problem" could mean a health problem for the individual whose sample is far away from the mean.